app/helpers/data_getter.py

```diff
@@ -216,16 +216,22 @@ def get_tracks_by_event_id():
         return Track.query.filter_by(event_id=get_event_id())
 
     @staticmethod
-    def get_sessions(event_id, state='accepted'):
+    def get_sessions(event_id, state=None):
         """
         :param state: State of the session
         :param event_id: Event id
         :return: Return all Sessions objects with Event id
         """
-        return Session.query.filter_by(
-            event_id=event_id,
-            state=state
-        ).filter(Session.deleted_at.is_(None))
+        if state is None:
+            return Session.query.filter_by(
```

Comment:
Why do you keep putting the `.` at the end and then using `\` to continue on the next line when the `.` can be put in the next line ? 

```python
return Session.query.filter_by(
    event_id=event_id
)
.filter(or_(Session.state == 'accepted', Session.state == 'confirmed')))
.filter(Session.deleted_at.is_(None))
```